app/config/mandatee-table-config.js

```diff
@@ -64,13 +73,19 @@ export const mandateeTableConfigIVGR = (meeting) => {
           ?mandataris org:holds ?mandaat.
           ?mandaat org:role <${BESTUURSFUNCTIE_CODES.GEMEENTERAADSLID}>.
 
-          ?bestuursorgaan org:hasPost ?mandaat.
-          ?bestuursorgaan lmb:heeftBestuursperiode <${BESTUURSPERIODES['2024-heden']}>.
+          ?bestuursorgaanIT org:hasPost ?mandaat.
+          ?bestuursorgaanIT lmb:heeftBestuursperiode <${BESTUURSPERIODES['2024-heden']}>.
+          ?bestuursorgaanIT mandaat:isTijdspecialisatieVan ?bestuursorgaan.
+          ?bestuursorgaan besluit:classificatie ?classificatie.
+          VALUES ?classificatie {
+            <${BESTUURSORGAAN_CLASSIFICATIE_CODES.GEMEENTERAAD}>
+          }
+          ?bestuursorgaan besluit:bestuurt <${bestuurseenheid.uri}>.
         }
       `;
         return executeQuery({
           query: sparqlQuery,
-          endpoint: '/vendor-proxy/query',
+          endpoint: '/raw-sparql',
```

Comment:
True, that would be better indeed. It does mean that (with the default config) updates can take in the worst case 600 seconds to come through. (That's the default TTL of the sparql-cache)

Reply:
it only does a quick query on pageload, and then only hits it on user click
I think instant updates will be nicer over the potential performance
especially since we actually _do_ expect the data to be different everytime this gets hit (otherwise, why are they clicking sync?)